Modules/_io/winconsoleio.c

```diff
@@ -133,6 +133,21 @@ char _PyIO_get_console_type(PyObject *path_or_fd) {
     return m;
 }
 
+static HANDLE py_get_osfhandle(int fd)
```

Comment:
`_Py_get_osfhandle`, `_Py_get_osfhandle_noraise`, `_Py_open_osfhandle`, and `_Py_open_osfhandle_noraise` could be added in Python/fileutils.c. These CRT functions are called in fileutils.c, msvcrtmodule.c, posixmodule.c, mmapmodule.c, myreadline.c, _testconsole.c, and here.